src/Features/Core/Portable/Wrapping/SeparatedSyntaxList/AbstractSeparatedSyntaxListWrapper.cs

```diff
@@ -10,29 +11,21 @@ namespace Microsoft.CodeAnalysis.Wrapping.SeparatedSyntaxList
     using Microsoft.CodeAnalysis.Indentation;
 
     /// <summary>
-    /// Base type for all wrappers that involve wrapping a comma-separated list of items.
+    /// Base type for all wrappers that involve wrapping a comma-separated list of arguments or parameters.
```

Comment:
good comment.  might be good to mention that it specifically does not apply to 'initializers' up to you though.

Reply:
I didn't add this for now because I couldn't think of a great way to work that into the comment. My first couple of attempts didn't seem great, so I decided to leave as is.